apps/staking/app/stake/[address]/NewStake.tsx

```diff
@@ -105,27 +107,51 @@ export function NewStake({ contract }: { contract: ContributionContract }) {
     },
   });
 
-  const form = useForm<StakeFormSchema>({
-    resolver: zodResolver(formSchema),
-    defaultValues: {
+  const defaultValues = useMemo(() => {
```

Comment:
Good to do this but why did you have to do it? Issues with react-form. Asking for my own learning. Feel free to resolve and let me know later